{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2023 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-Jinja2",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "The python-Jinja2 package was updated to version 2.7.3 to fix a security issues and some build problems. \n\nThe following vulnerabilities were fixed:\n- Update to 2.7.3 (bnc#858239, CVE-2014-0012)\n  - Security issue: Corrected the security fix for the cache folder.  \n    This fix was provided by RedHat.\n\nThe following build issues were fixed:\n- run testsuite during build\n- adjust dependency to use up to date package name for python-MarkupSafe\n- fix package build (file selection missing)\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "SUSE-SLE12-CLOUD-5-2015-363,SUSE-Storage-1.0-2015-363",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2015:1336-1",
				"url": "https://www.suse.com/support/update/announcement/2015/suse-su-20151336-1/"
			},
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2015_1336-1.json"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2015:1336-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-August/001522.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 858239",
				"url": "https://bugzilla.suse.com/858239"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2014-0012 page",
				"url": "https://www.suse.com/security/cve/CVE-2014-0012/"
			}
		],
		"title": "Security update for python-Jinja2",
		"tracking": {
			"current_release_date": "2015-05-11T09:09:44Z",
			"generator": {
				"date": "2015-05-11T09:09:44Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2015:1336-1",
			"initial_release_date": "2015-05-11T09:09:44Z",
			"revision_history": [
				{
					"date": "2015-05-11T09:09:44Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Jinja2-2.7.3-4.1.noarch",
								"product": {
									"name": "python-Jinja2-2.7.3-4.1.noarch",
									"product_id": "python-Jinja2-2.7.3-4.1.noarch"
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
								"product": {
									"name": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
									"product_id": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-sle12-cloud-compute:5"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 1.0",
								"product": {
									"name": "SUSE Enterprise Storage 1.0",
									"product_id": "SUSE Enterprise Storage 1.0",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:1.0"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Jinja2-2.7.3-4.1.noarch as component of SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
					"product_id": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5:python-Jinja2-2.7.3-4.1.noarch"
				},
				"product_reference": "python-Jinja2-2.7.3-4.1.noarch",
				"relates_to_product_reference": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Jinja2-2.7.3-4.1.noarch as component of SUSE Enterprise Storage 1.0",
					"product_id": "SUSE Enterprise Storage 1.0:python-Jinja2-2.7.3-4.1.noarch"
				},
				"product_reference": "python-Jinja2-2.7.3-4.1.noarch",
				"relates_to_product_reference": "SUSE Enterprise Storage 1.0"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2014-0012",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2014-0012"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "FileSystemBytecodeCache in Jinja2 2.7.2 does not properly create temporary directories, which allows local users to gain privileges by pre-creating a temporary directory with a user's uid. NOTE: this vulnerability exists because of an incomplete fix for CVE-2014-1402.",
					"title": "Vulnerability description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5:python-Jinja2-2.7.3-4.1.noarch",
					"SUSE Enterprise Storage 1.0:python-Jinja2-2.7.3-4.1.noarch"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2014-0012",
					"url": "https://www.suse.com/security/cve/CVE-2014-0012"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 858239 for CVE-2014-0012",
					"url": "https://bugzilla.suse.com/858239"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5:python-Jinja2-2.7.3-4.1.noarch",
						"SUSE Enterprise Storage 1.0:python-Jinja2-2.7.3-4.1.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2015-05-11T09:09:44Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2014-0012"
		}
	]
}
